contracts/scripts/fundAccounts.js

```diff
@@ -0,0 +1,11 @@
+const devAccounts = [
+  "0xf39Fd6e51aad88F6F4ce6aB8827279cffFb92266",
+  "0x70997970C51812dc3A010C7d01b50e0d17dc79C8",
+  "0x3C44CdDdB6a900fa2b585dd299e03d12FA4293BC",
+  "0x90F79bf6EB2c4f870365E785982E1f101E93b906",
+  "0x15d34AAf54267DB7D7c367839AAf71A00a2C6A65",
+];
+
+for (const to of devAccounts) {
+  eth.sendTransaction({ from: eth.coinbase, to, value: 1e22 });
+}
```

Comment:
Rather than doing this, I would recommend seeding the geth instance with the Hardhat accounts (https://geth.ethereum.org/docs/fundamentals/backup-restore). This will make it easier to switch between hardhat & geth at will.

Reply:
I'm not sure what you're suggesting here, could you be a bit more specific?

Based on the link, it sounds like the suggestion is to get a snapshot with those accounts (somehow? some other method?) and then restore from that snapshot. That seems a lot more complicated, so I'm guessing you meant something else. Perhaps you pasted the wrong link?